examples/plot_baby_afq.py

```diff
@@ -0,0 +1,92 @@
+"""
+==============================
+Pediatric Profiles:
+==============================
+
+The following is an example of tractometry for pediatric bundles.
+
+TODO cite @mareike
+
+.. note::
+
+  This example uses the Yeatman et al. waypoint ROI approach, first
+  described in [Yeatman2012]_ and further elaborated in [Yeatman2014]_.
+
+"""
+import os.path as op
+import plotly
+
+from AFQ import api
+import AFQ.data as afd
+
+from AFQ.definitions.mask import RoiMask, MaskFile
+
+##########################################################################
+# Get example data:
+# -----------------
+# **Diffusion dataset**
+#
+# .. note::
```

Comment:
We don't actually use any dataset here, so I would say that this is all somewhat theoretical, and link to the repo that contains the code for the dHCP data analysis, to say that's what we actually do.